argot.go

```diff
@@ -27,8 +27,12 @@ var (
 
 func formatFatalSteps(results Steps, err error) string {
 	msg := ""
-	if len(results) > 0 {
-		msg = "Achieved Steps:\n" + defaultConfig.Sprint(results) + "\n"
+	l := len(results)
+	if l > 1 {
+		msg = "Achieved Steps:\n" + defaultConfig.Sprint(results[:l-1]) + "\n"
+	}
+	if l > 0 {
+		msg = msg + "Failed Step: " + defaultConfig.Sprint(&results[l-1]) + "\n"
```

Comment:
For consistency with the previous, I'd add a \n at the end of the initial string.